doc/tutorial/concepts/custom_validator_solution1.cpp

```diff
@@ -0,0 +1,21 @@
+#include <seqan3/argument_parser/all.hpp>
+
+struct custom_validator
+{
+    using value_type = double; // used for all arithmetic types
+
+    void operator() (value_type const &) const
+    {
+        // add implementation later
+    }
+
+    std::string get_help_page_message () const
+    {
+        // add real implementation later
```

Comment:
```suggestion
        // add implementation later
```

Reply:
but there is an implementation, because there is a return statement